web3swift/KeystoreManager/Classes/BIP39.swift

```diff
@@ -11,10 +11,31 @@ import CryptoSwift
 
 public enum BIP39Language {
     case english
+    case chinese_simplified
+    case chinese_traditional
+    case japanese
```

Comment:
japanese language requires a special separator of seed words

Reply:
@yuezy thank you for your addition. as soon as you add separators - we would be glad to merge! 👍

Reply:
@yuezy we have to consider this in this commit:

>*Developers implementing phrase generation or checksum verification must separate words using ideographic spaces / accommodate users inputting ideographic spaces.* (UTF-8 bytes: 0xE38080; C/C+/Java: "\u3000"; Python: u"\u3000")
https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md#japanese